site/en/docs/extensions/mv3/devtools/index.md

```diff
@@ -212,21 +196,13 @@ chrome.devtools.inspectedWindow.eval(
 );
 ```
 
-Alternatively, use the `useContentScriptContext: true` option for `inspectedWindow.eval()` to
-evaluate the expression in the same context as the content scripts. Calling `eval` with
-`useContentScriptContext: true` does not _create_ a content script context, so you must load a
+You can also use the `useContentScriptContext: true` option for `inspectedWindow.eval()` to
+evaluate the expression in the same context as the content scripts. To use this option, you must load a
 context script before calling `eval`, either by calling `executeScript` or by specifying a content
-script in the `manifest.json` file.
-
-Once the context script context exists, you can use this option to inject additional content
-scripts.
+script in the `manifest.json` file. After the context script context loads, you can also use this option to
```

Comment:
```suggestion
in the `manifest.json` file. After the context script context loads, you can also use this option to
```

Reply:
This paragraph is more of a mess than I realized. Which ones should be "content" and which ones "context"?

Reply:
I don't know what I was thinking when I removed this. It was a mistake.